src/django_idom/hooks.py

```diff
@@ -65,17 +65,14 @@ def use_origin() -> str | None:
         return None
 
 
-def use_scope() -> dict[str, Any]:
+def use_scope() -> MutableMapping[str, Any]:
     """Get the current ASGI scope dictionary"""
-    return use_websocket().scope
+    return _use_scope()
 
 
-def use_websocket() -> IdomWebsocket:
-    """Get the current IdomWebsocket object"""
-    websocket = use_context(WebsocketContext)
-    if websocket is None:
-        raise RuntimeError("No websocket. Are you running with a Django server?")
-    return websocket
+def use_connection() -> Connection:
+    """Get the current `Connection` object"""
+    return _use_connection()  # type: ignore
```

Comment:
Should these hooks be deprecated in favor of the hooks from core?

Reply:
I tried that out initially, and it felt wrong. The documentation for how those specific hooks behave with Django-IDOM are going to be long in this repo. Feels weird to then have the user's import path be in core.

Plus `use_origin` wouldn't be in core, so that's yet another mismatch.